Sources/MapboxMaps/Gestures/CameraManagerProtocol.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+internal protocol CameraManagerProtocol: AnyObject {
```

Comment:
Nit: Can we group `CameraManagerProtocol` in the file path for Camera, not Gestures

Reply:
Lets do this afterwards.

Reply:
But its the protocol on the camera, like it belongs to the camera from what I see.